Gemfile

```diff
@@ -1,56 +1,67 @@
-source 'https://rubygems.org'
+source "https://rubygems.org"
+# ruby '2.1.4'
```

Comment:
Eu manteria a versão do ruby (`ruby '2.2.0'`)  aqui pois o heroku utiliza a versão do ruby especificada no gemfile.
https://devcenter.heroku.com/articles/ruby-versions#selecting-a-version-of-ruby